<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="ros_tracked_robot">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.0130734817648772 0.0172872819635437 0.0297178850760488"
        rpy="0 0 0" />
      <mass
        value="1.58749693033699" />
      <inertia
        ixx="0.00418899780402248"
        ixy="-1.0689693830114E-18"
        ixz="7.47701295687736E-21"
        iyy="0.00633806354929637"
        iyz="3.4254676519062E-19"
        izz="0.00973422182190228" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="arm0">
    <inertial>
      <origin
        xyz="0.10082 0.05557 0.079975"
        rpy="0 0 0" />
      <mass
        value="0.39827" />
      <inertia
        ixx="0.00063449"
        ixy="-1.3116E-05"
        ixz="-8.3043E-09"
        iyy="0.0009933"
        iyz="8.5566E-07"
        izz="0.00064644" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/arm0.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.0078431 0.23922 0.82353 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/arm0.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="arm0_joint"
    type="fixed">
    <origin
      xyz="-0.11307 0.096556 0.0045234"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="arm0" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="arm1">
    <inertial>
      <origin
        xyz="-0.021405 0.070686 0.022815"
        rpy="0 0 0" />
      <mass
        value="0.11467" />
      <inertia
        ixx="0.00033066"
        ixy="0.00010773"
        ixz="-8.842E-07"
        iyy="5.3066E-05"
        iyz="2.8664E-06"
        izz="0.00037856" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/arm1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.0078431 0.23922 0.82353 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/arm1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="arm_1_joint"
    type="revolute">
    <origin
      xyz="0.1 0.14 0.07"
      rpy="0 0 0" />
    <parent
      link="arm0" />
    <child
      link="arm1" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="arm2">
    <inertial>
      <origin
        xyz="-0.064519 -0.021892 -0.012108"
        rpy="0 0 0" />
      <mass
        value="0.097049" />
      <inertia
        ixx="5.1899E-05"
        ixy="-0.0001056"
        ixz="-2.4558E-06"
        iyy="0.00028921"
        iyz="-8.3328E-07"
        izz="0.00033478" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/arm2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.0078431 0.23922 0.82353 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/arm2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="arm_2_joint"
    type="revolute">
    <origin
      xyz="-0.056769 0.16024 0.05"
      rpy="0 0 0" />
    <parent
      link="arm1" />
    <child
      link="arm2" />
    <axis
      xyz="0 0 -1" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="imu">
    <inertial>
      <origin
        xyz="0 0.0075 0"
        rpy="0 0 0" />
      <mass
        value="0.0045" />
      <inertia
        ixx="1.6875E-07"
        ixy="-3.2843E-40"
        ixz="7.3802E-24"
        iyy="2.3438E-07"
        iyz="-3.3087E-24"
        izz="2.3438E-07" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/imu.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/imu.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="imu_joint"
    type="fixed">
    <origin
      xyz="-0.013073 0.016556 0.0045234"
      rpy="1.5708 0 3.1416" />
    <parent
      link="base_link" />
    <child
      link="imu" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="camera">
    <inertial>
      <origin
        xyz="-0.0075 -0.0075 0"
        rpy="0 0 0" />
      <mass
        value="0.0045" />
      <inertia
        ixx="2.3438E-07"
        ixy="1.5247E-24"
        ixz="-2.7948E-24"
        iyy="2.3438E-07"
        iyz="4.4116E-24"
        izz="1.6875E-07" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/camera.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/camera.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="camera_joint"
    type="fixed">
    <origin
      xyz="-0.11307 0.016556 0.069523"
      rpy="1.5708 0 3.1416" />
    <parent
      link="base_link" />
    <child
      link="camera" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="wheel1">
    <inertial>
      <origin
        xyz="1.3878E-17 0 0.015"
        rpy="0 0 0" />
      <mass
        value="0.094248" />
      <inertia
        ixx="3.3281E-05"
        ixy="5.7779E-21"
        ixz="4.2558E-22"
        iyy="3.3281E-05"
        iyz="1.5415E-21"
        izz="5.2425E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.24706 1 0.55294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel1_joint"
    type="continuous">
    <origin
      xyz="0.036927 0.12656 0.019523"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel1" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="wheel2">
    <inertial>
      <origin
        xyz="-2.7756E-17 3.4694E-18 0.015"
        rpy="0 0 0" />
      <mass
        value="0.094248" />
      <inertia
        ixx="3.3281E-05"
        ixy="4.5675E-21"
        ixz="1.0372E-21"
        iyy="3.3281E-05"
        iyz="1.9191E-21"
        izz="5.2425E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.24706 1 0.55294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel2_joint"
    type="continuous">
    <origin
      xyz="-0.063073 0.12656 0.019523"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel2" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="wheel3">
    <inertial>
      <origin
        xyz="2.7756E-17 -1.7347E-17 -0.015"
        rpy="0 0 0" />
      <mass
        value="0.094248" />
      <inertia
        ixx="3.3281E-05"
        ixy="5.4413E-22"
        ixz="-9.7795E-22"
        iyy="3.3281E-05"
        iyz="1.5882E-21"
        izz="5.2425E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.24706 1 0.55294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel3_joint"
    type="continuous">
    <origin
      xyz="-0.063073 -0.093444 0.019523"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel3" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="wheel4">
    <inertial>
      <origin
        xyz="1.3878E-17 -1.7347E-17 -0.015"
        rpy="0 0 0" />
      <mass
        value="0.094248" />
      <inertia
        ixx="3.3281E-05"
        ixy="-1.3634E-21"
        ixz="-5.6741E-24"
        iyy="3.3281E-05"
        iyz="-5.4256E-22"
        izz="5.2425E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.24706 1 0.55294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/wheel4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel4_joint"
    type="continuous">
    <origin
      xyz="0.036927 -0.093444 0.019523"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel4" />
    <axis
      xyz="0 0 1" />
    <limit
      lower="0"
      upper="0"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="tracked_right">
    <inertial>
      <origin
        xyz="0.018463 -0.0335 -5.5511E-17"
        rpy="0 0 0" />
      <mass
        value="0.01222" />
      <inertia
        ixx="1.0839E-05"
        ixy="3.6965E-20"
        ixz="3.6125E-21"
        iyy="3.7924E-05"
        iyz="-2.2037E-21"
        izz="4.693E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/tracked_right.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.29804 0.29804 0.29804 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/tracked_right.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="tracked_right_joint"
    type="fixed">
    <origin
      xyz="-0.031537 0.11156 0.053023"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="tracked_right" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="tracked_left">
    <inertial>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <mass
        value="0" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/tracked_left.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://ros_tracked_robot/meshes/tracked_left.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="tracked_left_joint"
    type="fixed">
    <origin
      xyz="-0.031537 -0.078444 0.053023"
      rpy="1.5708 0 0" />
    <parent
      link="base_link" />
    <child
      link="tracked_left" />
    <axis
      xyz="0 0 0" />
  </joint>
</robot>